### ISSUE TITLE
Bug: Tweening - EasingFunctions

### ISSUE THREAD

**Issue by DerMeister** [craftworkgames/MonoGame.Extended#456]
The power values for the cubic and quadratic easing functions are swapped.

``` Java
        public static float CubicEaseIn(float value) => Power.EaseIn(value, 2);
        public static float CubicEaseOut(float value) => Power.EaseOut(value, 2);
        public static float CubicEaseInOut(float value) => Power.EaseInOut(value, 2);

        public static float QuadraticEaseIn(float value) => Power.EaseIn(value, 3);
        public static float QuadraticEaseOut(float value) => Power.EaseOut(value, 3);
        public static float QuadraticEaseInOut(float value) => Power.EaseInOut(value, 3);
```

**Comment by craftworkgames** [comment]
Right. Nicely spotted.

Would you like to have a go at making a PR to fix it?

**Comment by craftworkgames** [comment]
Fixed with PR #457
